content/en/docs/instrumentation/java/automatic/spring-boot.md

```diff
@@ -23,6 +23,84 @@ Spring Boot starter, see
 
 ## Configuration
 
+### Bill of Materials
```

Comment:
```suggestion
### Dependency management
```

Reply:
I would prefer to keep "Bill of Materials" because the next section is  "OpenTelemetry Starter dependency"

Reply:
my thought is that someone who doesn't know what Bill Of Materials are may skip over the section and go straight to "OpenTelemetry Starter dependency"